MAUI/DigitalGauge/Character-Display-Types.md

```diff
@@ -34,6 +34,9 @@ Numbers can be displayed in a digital gauge in different formats using the `Char
 
 {% endtabs %}
 
+![Display Number](Images\DigitalGauge9.png)
```

Comment:
reference like this

[tooltip-template-in-maui-stepprogressbar]

Reply:
Updated Changes